ipython/find_parameter_sources_and_assign_uncertainties.ipynb

```diff
@@ -25,6 +25,31 @@
     "__Note__: The RMG-database version must match the version used to generate the model. RMG will attempt to recreate the kinetics estimate for each reaction and may fail if the database is different."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run RMG job"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "!python ../rmg.py ./data/parse_source/input.py"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Perform parameter uncertainty assignment"
+   ]
+  },
```

Comment:
I agree with the proposal for deleting those model jobs which may have incompatibility issues with the newest version. Just a minor comments. Can you add a comment/instruction to say this cell is only used when people want to try and learn this module with a toy model.

Reply:
@xiaoruiDong Thanks for your review and comment! I've added the a short instruction in the markdown cell above this cell to inform the user when to use this cell.